standalone-metastore/metastore-server/src/main/sql/derby/upgrade-4.0.0-alpha-2-to-4.0.0-beta-1.derby.sql

```diff
@@ -29,5 +29,21 @@ CREATE INDEX "APP"."PCS_STATS_IDX" ON "APP"."PART_COL_STATS" ("DB_NAME","TABLE_N
 -- HIVE-27186
 ALTER TABLE "APP"."METASTORE_DB_PROPERTIES" ADD PROPERTYCONTENT BLOB;
 
+-- HIVE-27457
+UPDATE SDS
```

Comment:
some q test failed, I have attached the link, can you please take a look at those.